shared/middleware/helpers/storage_apps.rb

```diff
@@ -14,7 +14,7 @@ def initialize(storage_id)
   def create(value, ip:, type: nil, published_at: nil)
     timestamp = DateTime.now
     row = {
-      storage_id: @storage_id,
+      storage_id: @storage_id.to_i,
```

Comment:
`@storage_id` should be an integer, so I'd debug back up the call stack from there to see how it ended up being an encrypted channel id such as 'de96e1f00e331d46db5235f93d5dadbf'